### ISSUE TITLE
Still the same error :(

### ISSUE THREAD

**Issue by pookie1** [nwoolls/MultiMiner#335]
I downloaded bfgminer and manually extracted it to my multiminer/miners/ 

When I run multiminer it still tries to dload bfgminer and then says the same error for bfgminer.exe

Here is a  pic of the bfgminer manually downloaded and extracted. And shows what happens when I try to run Multiminer.

I must be really missing somethiong. I've spent hours on this ... 

![screenshot 24](https://user-images.githubusercontent.com/35513067/35090102-d41a1bec-fc8c-11e7-90d7-1a7221ecb1c4.png)

**Comment by auroracoin** [comment]
Same thing here..... for Windows 8.1 nvidia cards.

**Comment by nwoolls** [comment]
@pookie1 *please* stop opening new issues - this is the 3rd issue you've opened about the same problem. Just update your existing issues.

As I said in previous messages, extract BFGMiner to `<MultiMiner>\Miners\BFGMiner` - check the casing.

**Comment by pookie1** [comment]
My apologies for opening 3 new issues.

 I wasn't aware that it was case sensitive. I will try as you suggested. 

Thanks.

**Comment by nwoolls** [comment]
@pookie1 also check inside the `BFGMiner` folder after you have unzipped and ensure there is a `bfgminer.exe` inside. If not, you still have some form of antivirus blocking crypto miners.

**Comment by pookie1** [comment]
I have changed the case to BFGMiner and confirmed there is bfgminer.exe in the folder.

**Comment by pookie1** [comment]
now when I run MultiMiner.Win.exe nothing happens :(

**Comment by pookie1** [comment]
Ok, now Multiminer opened and told me there is no BFGMiner. So I checked the folder and the .exe file is gone. It was there a few moments ago. So I started over again and unzipped the BFGMiner file to the correct path and confirmed the BFGMiner.exe file was there. I ran Multiminer.win.exe and once again it loaded and opened the main window but then again told me there is no BFGMiner detected :( Arghhhhh

**Comment by JonVielhaber** [comment]
Check all of your anti-virus and malware products. They are flagging all miners as unwanted programs. Since it is disappearing after it's started, my guess would be it's MalwareBytes if you have that. Anti-virus usually prevents you from unzipping it in the first place. MalwareBytes picks it up during an execution attempt.

**Comment by pookie1** [comment]
![screenshot 32](https://user-images.githubusercontent.com/35513067/35102537-4e706f70-fcb7-11e7-9e88-60b01db9ac69.png)

This is a new error I am getting now.

**Comment by nwoolls** [comment]
@pookie1 you need to fix your anti-virus situation. You will not have success doing any crypto mining until then.

**Comment by pookie1** [comment]
I have disabled everything I can think of in Windows Defender settings and in McAffe. They are the only 2 virus/malware programs I know I have. I have not added any since I bought the PC a few months ago. They came with the PC. 

I don't have Malwarebytes.

I have today successfully installed and am running NiceHash Miner. (see screenshot). But it's only BTC Miner, which is useless for GPU Mining. I only tried it for a test.

![screenshot 36](https://user-images.githubusercontent.com/35513067/35103251-6d33027c-fcb9-11e7-8e8f-bb51de84831e.png)


Not having much luck am I :(

I do appreciate your help though very much.

**Comment by pookie1** [comment]
Hi, as a last resort I have now uninstalled my McAfee. But still no difference ...arghhhhh

**Comment by JonVielhaber** [comment]
Check your program files list for MalwareBytes, Avast, PCProtect, or anythingelse . Also, look at theicons  down by your clock . If it lets you uncompress the file, but it deletes it when you run it, that's most likely an anti-malware program. It's not a problem with MultiMiner. You can also try running it from the command line. You might get a better error message.

**Comment by pookie1** [comment]
Hi, I have checked for any virus/malware programs...nothing. I tried from the CMD = nothing. Now when I select multiminer.win.exe to run it does nothing :(

**Comment by pookie1** [comment]
It's like I'm cursed LOL

I have uninstalled my only antivirus (McAffee). The only protection left was Windows Defender, of which I have disabled every setting I could find.

I have installed/uninstalled Multiminer and BFGminer a dozen times, and tried to run it in various ways, as per advice. It just doesn't want to install properly.

Such a shame. You are probably correct that it is a problem my end. But I am not that familiar with Windows to properly diagnose and fix the issue.

This is my 3rd night trying to resolve this :\ 

I am grateful for the help though.
